buku

```diff
@@ -1614,17 +1647,21 @@ if __name__ == '__main__':
             new_url = ''
 
         append = False
+        delete = False
         if tagManual is not None:
-            if tagManual[0] == '+' and len(tagManual) == 1:
+            if (tagManual[0] == '+' or tagManual[0] == '-') and len(tagManual) == 1:
                 tagManual = None
             elif tagManual[0] == '+':
                 tagManual = tagManual[1:]
                 append = True
+            elif tagManual[0] == '-':
+                tagManual = tagManual[1:]
+                delete = True
 
         # Parse tags into a comma-separated string
         tags = parse_tags(tagManual)
 
-        bdb.update_bookmark(update_index, new_url, titleManual, tags, description, append)
+        bdb.update_bookmark(update_index, new_url, titleManual, tags, description, append, delete)
 
     # Delete record(s)
     if args.delete is not None:
```

Comment:
Please update the program help and man page (refer to with append related sections).